peewee_async/__init__.py

```diff
@@ -43,4 +44,6 @@
 
 register_database(PooledPostgresqlDatabase, 'postgres+pool+async', 'postgresql+pool+async')
 register_database(PooledPostgresqlExtDatabase, 'postgresext+pool+async', 'postgresqlext+pool+async')
+register_database(PooledPsycopg3PostgresqlDatabase, 'postgres+psycopg+pool+async', 'postgres+psycopg+pool+async')
```

Comment:
Let's name just Psycopg3Database. We will rename PooledPostgresqlDatabase to AioPgDatabase in future. I believe we also have to have Psycopg3ExtDatabase